src/event-helper.js

```diff
@@ -152,18 +153,32 @@ export function loadPromise(eleOrWindow) {
     return Promise.resolve(eleOrWindow);
   }
   const loadingPromise = new Promise((resolve, reject) => {
+    const isMediaElement = isHTMLMediaElement(eleOrWindow);
     // Listen once since IE 5/6/7 fire the onload event continuously for
     // animated GIFs.
-    const {tagName} = eleOrWindow;
-    if (tagName === 'AUDIO' || tagName === 'VIDEO') {
-      unlistenLoad = listenOnce(eleOrWindow, 'loadstart', resolve);
+    if (isMediaElement) {
+      // The following event can be triggered by the media or one of its
+      // sources. Using capture is required as the media events do not bubble.
```

Comment:
📝I was looking for a source for this and it looks like `loadedmetadata` ([1]) and `error` ([2]) and some others are *simple events* which are defined to not bubble ([3])

[1]: https://www.w3.org/TR/2011/WD-html5-20110113/video.html#fire-loadedmetadata
[2]: https://www.w3.org/TR/2011/WD-html5-20110113/video.html#fatal-decode-error
[3]: https://www.w3.org/TR/2011/WD-html5-20110113/webappapis.html#fire-a-simple-event